storagemarket/impl/shardregistration.go

```diff
@@ -9,25 +9,16 @@ import (
 
 	"github.com/filecoin-project/dagstore"
 	"github.com/filecoin-project/go-address"
-	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/go-statemachine/fsm"
-	"github.com/filecoin-project/lotus/chain/actors/builtin/miner"
-	"github.com/filecoin-project/lotus/chain/types"
-	"github.com/filecoin-project/lotus/extern/sector-storage/storiface"
-	"github.com/filecoin-project/specs-storage/storage"
 
 	mktdagstore "github.com/filecoin-project/go-fil-markets/dagstore"
+	"github.com/filecoin-project/go-fil-markets/piecestore"
 	"github.com/filecoin-project/go-fil-markets/storagemarket"
 	"github.com/filecoin-project/go-fil-markets/storagemarket/impl/providerstates"
 )
 
 var shardRegKey = datastore.NewKey("shards-registered")
 
-type SectorStateAccessor interface {
-	StateSectorGetInfo(context.Context, address.Address, abi.SectorNumber, types.TipSetKey) (*miner.SectorOnChainInfo, error)
-	IsUnsealed(ctx context.Context, sector storage.SectorRef, offset storiface.UnpaddedByteIndex, size abi.UnpaddedPieceSize) (bool, error)
-}
-
```

Comment:
nit: I actually preferred @dirkmc's method of tightening the interface requirement to only the 2 methods that are needed. It's more idiomatic and specific ([defining interface at the point of use](https://github.com/golang/go/wiki/CodeReviewComments#interfaces)). This new approach leaks, in sense that now the `ShardMigrator` is now privy to the fact that these two behaviours are provided by two different system components.

That said, no need to revert this at all, it's absolutely not a big deal here because this component has a limited lifetime and is not reusable.